people-ops/time-off-and-overtime.html.md

```diff
@@ -77,19 +74,42 @@ and so on.
 
 ## Sick leave
 
-For communications regarding sick leave, employees must contact their physician to request a sick 
-leave certificate. The physician will send the certificate electronically to INPS, which, in turn, 
-will make it available in electronic form. Based on the sick leave days granted by the attending 
-physician, the employee must  communicate to the company the days of absence together with the
+We support you in the unfortunate case of sick leave. We pay for sick days a different percentage of
```

Comment:
The second sentence here seems to me to be a bit off on the structure, `we pay #{modifiers} for sick days` sounds more correct to me, like:
`We pay a different percentage of the daily pay for sick days, depending...`

Reply:
Thanks!